src/test/java/TestJsonMapper.java

```diff
@@ -0,0 +1,34 @@
+import Mappers.JSONMapper;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import services.FetchToTheForexAPI;
+
+import java.nio.channels.AsynchronousServerSocketChannel;
+import java.util.Objects;
+
+public class TestJsonMapper {
```

Comment:
If String Json = null or maybe "abc"?

Reply:
I see I need to do an additional test :)